podkrepi-bg/frontend
# bugfix/1950-Financial-Report-2023-Typo

Replaced Financial_Report_2023.pdf file with the updated one without the typos

Closes [#1950](https://github.com/podkrepi-bg/frontend/issues/1950)

## Motivation and context







## Screenshots:

![image](https://github.com/user-attachments/assets/f19fb599-6c01-42ca-91b9-428176dbce3f)

## Testing

### Steps to test

Steps to test

1. Open https://podkrepi.bg/finance-reports/Podkrepi.bg_Financial_Report_2023.pdf
2. Scroll to page 24 of the pdf document
3. Ensure typos are corrected

### Affected urls







## Environment

### New environment variables:




### New or updated dependencies:



Dependency name|Previous version|Updated version|Details
---|---|---|---
`dependency/name`|`v1.0.0`|`v2.0.0`|
